packages/formstr-app/src/containers/CreateFormNew/components/AnswerSettings/RightAnswer.tsx

```diff
@@ -5,40 +5,61 @@ import { Tooltip, Typography } from "antd";
 const { Text } = Typography;
 
 interface RightAnswerProps {
-  answerType: AnswerTypes;
-  answerSettings: AnswerSettings;
-  onChange: (answer: string) => void;
+  answerType: AnswerTypes | string;
```

Comment:
why is this `AnswerTypes | string` ? in what case would it not be an AnswerType?